tide_media.module

```diff
@@ -351,7 +351,7 @@ function tide_media_entity_embed_alter(array &$build, EntityInterface $entity, a
  * Custom validation.
  */
 function tide_media_form_media_embedded_video_validate($form, FormStateInterface $form_state) {
-  if ($form_state->hasValue(['inline_entity_form'])) {
+  if ($form['widget']['inline_entity_form']['#bundle'] === 'embedded_video') {
```

Comment:
Hi @vincent-gao, I have made the change to ensure validation on embedded_video page.

Reply:
please check if `$form['widget']['inline_entity_form']['#bundle']` exists first. 
it could be isset($form['widget']['inline_entity_form']['#bundle'])